src/main/java/io/codelex/flightplanner/controller/AirportController.java

```diff
@@ -0,0 +1,29 @@
+package io.codelex.flightplanner.controller;
+
+import io.codelex.flightplanner.model.Airport;
+import io.codelex.flightplanner.service.AirportServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class AirportController {
+
+    private final AirportServiceImpl airportServiceImpl;
```

Comment:
Te vajadzētu likt AirportService nevis AirportServiceImpl - viņš pats atradīs, ka tam interfeisam ir viena implementācija un to te pados.